tetris.py

```diff
@@ -111,8 +112,10 @@ def __init__(self, canvas, width, height, outline_color, current_block_center, s
         self.block_letter = random.choice(block_letters)
         self.rotate_counter = 0
         self.small_board = small_board
+        self.topbar_score = topbar_score
         self.draw_board()
         self.draw_block()
+        self.initial_score = 0
```

Comment:
Why is this called `initial_score`? It is not always the initial score, it changes when the game is played.

Reply:
Changed to score

Reply:
doesn't seem like it? please push

Reply:
Done